Assets/MarkerMetro/Editor/MarkerMetroBuilder.cs

```diff
@@ -81,13 +81,43 @@ public static void DoBuildUniversal (string outputPath)
             {
                 throw new Exception(error); // ensures exit code != 0.
             }
+#if UNITY_EDITOR
+            //Unity creates unused assets no matter what
+            //this hack removes those assets after the compilation
+            RemoveIncorrectAssets(outputPath);
+#endif
         }
+        static void RemoveIncorrectAssets(string outputPath)
+        {
+            string projectName = PlayerSettings.productName;
+            string assetsPath = outputPath + "/" + projectName + "/" + projectName + ".Windows/Assets/";
+
+            //removing unused assets for windows
+            if (File.Exists(assetsPath + "WideTile.scale-100.png"))
```

Comment:
agree with roberto here, you could clean up the code here by just calling File.Delete